src/test/java/nl/tudelft/semgroup4/GameStateControllerTest.java

```diff
@@ -0,0 +1,139 @@
+package nl.tudelft.semgroup4;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import nl.tudelft.model.Countdown;
+import nl.tudelft.model.Game;
+import nl.tudelft.semgroup4.resources.ResourcesWrapper;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.Input;
+import org.newdawn.slick.SlickException;
+import org.newdawn.slick.Sound;
+import org.newdawn.slick.state.StateBasedGame;
+
+public class GameStateControllerTest {
+    
+    private GameStateController gsController;
+    private Game mockedGame;
+    
+    @Before
+    public void setUp() {
+        mockedGame = Mockito.mock(Game.class);
+        gsController = new GameStateController(mockedGame);
+    }
+    
+    @Test
+    public void testConstructor() {
+        assertEquals(mockedGame, gsController.getGame());
+    }
+    
+    @Test
+    public void testTogglePauseMenu1() {
```

Comment:
More descriptive test cases would help, what do each of these tests assert?
E.g. <code>testTogglePauseMenuOpenWhenClosed()</code>

< naming